indexing-hadoop/src/main/java/io/druid/indexer/DeterminePartitionsJob.java

```diff
@@ -193,7 +193,7 @@ public boolean run()
       dimSelectionJob.setNumReduceTasks(config.getGranularitySpec().bucketIntervals().get().size());
       JobHelper.setupClasspath(
           JobHelper.distributedClassPath(config.getWorkingPath()),
-          JobHelper.distributedClassPath(config.makeIntermediatePath()),
+          JobHelper.distributedClassPath(new Path(config.makeIntermediatePath(), "dimSelection")),
```

Comment:
can we replace "dimSelection" by "determinePartitionsJob"? this can be same as above because "detetion" in JobHelper.runJobs() will only happen when both jobs are done and this will prevent uploading the same jar files twice. (actually that is the only reason "existing" exists in the JobHelper or else its useless).